_posts/blog/2019-02-15/2019-02-15-how-to-market-your-store-and-listings-on-openbazaar.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: "How To <span class='post-title-extra'>Market Your Store and Listings</span> on OpenBazaar"
+layout: post
+date: '2019-02-15 08:30:00 -0600'
+social_title: 'How To Market Your Store and Listings on OpenBazaar'
+social_description: 'Some of the earliest feedback we heard from ecommerce sellers who were interested in OpenBazaar was that they wanted a way to easily share their store and listings on their social networks. Here is how we did that.'
+---
+
+The shift to peer-to-peer online commerce won’t happen overnight. Until this technology becomes widespread, we need tools that help the transition go more smoothly.
+
+Some of the earliest feedback we heard from ecommerce sellers who were interested in OpenBazaar was that they wanted a way to easily share their store and listings on their social networks. To answer this we built [OpenBazaar.com](https://openbazaar.com).
+
+#### A regular website to look into a decentralized network
+
+[OpenBazaar](https://openbazaar.org/download) is an application that users have to download and run on their own computers to connect to and participate in the network. It is built this way on purpose because of the decentralized nature of the network. While the tech isn’t available for a full OpenBazaar buying & selling experience in a browser yet, that doesn’t mean there can’t be just a simple website that looks into the marketplace.
```

Comment:
I'd drop the "on purpose", I don't think it's needed.

Reply:
You might make it:

`It is built this way to make the decentralization of the network possible.`

Reply:
Done